src/components/impl/gumlet-image.js

```diff
@@ -36,10 +37,19 @@ export function GumletImage(props) {
 
   useEffect(loadGumlet);
 
+  useEffect(()=> {
+    imageEl.current.addEventListener('error',()=> {
```

Comment:
Use the browser error event itself, you need not add an event listener.

This would image performance as each image element has an event added by us.